client/settings/dpcs-settingspanel/panel_evoker.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [[ "$BASH_SOURCE" == "$0" ]]
+then 
+	sudo -s python2 settingspanel.py
```

Comment:
Note that one of the previous commits didn't use sudo, and the program crashed because it didn't have permission to `/etc/dpcs.conf`